libraries/AP_Param/tools/eedump_apparam.c

```diff
@@ -119,8 +119,8 @@ main(int argc, char *argv[])
     index = sizeof(*header);
     for (;; ) {
         uint8_t size;
-        const uint16_t key = ((uint16_t)var->key_high)<<8 | var->key_low;
         var = (struct Param_header *)&eeprom[index];
+        const uint16_t key = ((uint16_t)var->key_high)<<8 | var->key_low;
```

Comment:
This bugfix wasn't pulled out as a fix anywhere..

Reply:
i've removed that patch, and I think @hendjoshsr71 will patch it separately